Dockerfile

```diff
@@ -166,10 +166,18 @@ ARG INSTALL_PROVIDERS_FROM_SOURCES="false"
 # But it also can be `.` from local installation or GitHub URL pointing to specific branch or tag
```

Comment:
Should we add information about frontend?
```
# syntax = docker/dockerfile:1.3
```

Reply:
> Should we add information about frontend?

Good idea: 

> To use the external frontend, the first line of your Dockerfile needs to be # syntax=docker/dockerfile:1.3 pointing to the specific image you want to use.

> BuildKit also ships with Dockerfile frontend builtin but it is recommended to use an external image to make sure that all users use the same version on the builder and to pick up bugfixes automatically without waiting for a new version of BuildKit or Docker engine.

Reply:
Added